schema-extended/arxiv-extended.json

```diff
@@ -0,0 +1,623 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "arXiv Document Metadata",
+    "description": "Extended metadata record for an arXiv document.",
+    "type": "object",
+    "additionalProperties": false,
+
+    "definitions": {
+      "abstracts": {
+        "title": "List of abstracts",
+        "description": "The English and non-English versions of the abstract. The English abstract must be provided first.",
+        "type": "array",
+        "items": [
+          {
+            "type": "object",
+            "properties": {
+              "abstract": {
+                "type": "string",
+                "minLength": 1
+              },
+              "language": {
+                "const": "en"
+              }
+            },
+            "additionalProperties": false
+          }
+        ],
+        "additionalItems": {
+          "type": "object",
+          "properties": {
+            "abstract": {
+              "type": "string",
+              "minLength": 1
+            },
+            "language": {
+               "allOf": [
+                 {"$ref": "#/definitions/language"},
+                 {"not": {"enum": ["en"]}}
+             ]
+            }
+          },
+          "additionalProperties": false
+        }
+      },
+      "affiliation": {
+        "title": "Affiliation",
+        "description": "Institutional affiliation.",
+        "type": "object",
+        "properties": {
+          "name": {
+            "description": "The complete name of the affiliation.",
+            "type": "string",
+            "minLength": 1
+          },
+          "ror_id": {
+            "title": "ROR ID",
+            "description": "Research Organization Registry ID.",
+            "type": "string",
+            "pattern": "^https://ror\\.org/0\\w{6}\\d{2}$",
+            "examples": [
+              "https://ror.org/05bnh6r87",
+              "https://ror.org/05gzmn429",
+              "https://ror.org/01ggx4157",
+              "https://ror.org/03c3r2d17"
+            ]
+          }
+        },
+        "required": [
+          "name"
+        ],
+        "additionalProperties": false
+      },
+      "category": {
+        "title": "arXiv category",
+        "description": "This is the abbreviated form of the arXiv category using the category identifier",
+        "type": "string",
+        "pattern": "^([a-z]{3,}-[a-z]{2,}|[a-z]{2,}-?[a-z]+\\.[A-Z]{2})$",
+        "$comment": "TODO: enumeration"
+      },
+      "content_type": {
```

Comment:
content_type is too close to HTTP Content-Type. Some options: type_of_work literature_type paper_type